### PR TITLE
Mascots Commission Pages

### PR DESCRIPTION
**Pull request by mansona**
Fixes https://github.com/ember-learn/ember-website/issues/444

This PR really falls into the "not great, but good enough for now" category 🙈 It is no worse than current production, and actually fixes a bug on https://emberjs.com/mascots/payment/ 😂

### CODE REVIEW

File: app/styles/legacy/form.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+.tomster-form .inline-field input[type=text] {
+  width: auto
+}
+
+.tomster-form .inline-field label {
+  line-height: 1.6em
+}
+
+.hidden{display:none}
```
Comment (MelSumner) [inline app/styles/legacy/form.css]: we def shouldn't have hidden or clearfix in the website css. if we need these we need to put them in ember-styleguide, and then have a test that checks for duplicates.

File: app/templates/mascots/payment-sent.hbs [hunk 2]
```diff
@@ -1,9 +1,6 @@
-<h1>Tomster and Zoey Commission Payments</h1>
-
-<div class="tomster-payment section">
-  <div class="left-col">
-    <p>Thanks much! Looking forward to getting to work :)</p>
-    <p><a href="/mascots">&larr; Back to the other Tomsters and Zoeys</a></p>
-  </div>
-</div>
+<section class="container">
+  <h1>Tomster and Zoey Commission Payments</h1>
```
Comment (MelSumner) [inline app/templates/mascots/payment-sent.hbs]:
```suggestion
  <h1 id="commision-payments">Tomster and Zoey Commission Payments</h1>
```

File: app/templates/mascots/payment-sent.hbs [hunk 3]
```diff
@@ -1,9 +1,6 @@
-<h1>Tomster and Zoey Commission Payments</h1>
-
-<div class="tomster-payment section">
-  <div class="left-col">
-    <p>Thanks much! Looking forward to getting to work :)</p>
-    <p><a href="/mascots">&larr; Back to the other Tomsters and Zoeys</a></p>
-  </div>
-</div>
+<section class="container">
```
Comment (MelSumner) [inline app/templates/mascots/payment-sent.hbs]:
```suggestion
<section class="container" aria-labelledby="commision-payments">
```

File: app/styles/legacy/form.css (hunk 1 above)
Comment (mansona) [reply app/styles/legacy/form.css]: I've updated these to a) make sure they are scoped to only these wufoo forms and b) that they have a comment making sure it's clear that they are legacy and need to be deleted later

Essentially we need to properly sit down and design these pages, and there should **never** be a clearfix class helper in the website or the styleguide if we're doing things right. Also this `.hidden` class seems to be some sort of security feature to protect a bit against bots 🤔
